Lib/gftools/builder/recipeproviders/googlefonts.py

```diff
@@ -333,8 +333,7 @@ def build_a_static(self, source: File, instance: InstanceDescriptor, output):
         steps = [
             {"source": source.path},
         ]
-        # if we're running fontc we skip conversion to UFO
-        if not source.is_ufo and not self.config.get("use_fontc", False):
```

Comment:
if we remove this I believe we can also remove the place where we add the `use_fontc` flag to the config? it felt very hacky, and I believe it was only used in this one place.